composer.json

```diff
@@ -18,7 +18,7 @@
     "require-dev": {
         "phpunit/phpunit": "^9.5",
         "laminas/laminas-test": "^4.0.0",
-        "laminas/laminas-component-installer": "^3.2",
+        "laminas/laminas-component-installer": "^2.5",
```

Comment:
Downgrade?

Reply:
Originally I just had 8.0 and 8.1 for the security and test ci workflows which were complaining about this version (2.5) being outdated, now that I've added 7.4 it complains about the 3.2 version being too recent, the original version of this on the gitlab repo was 2.5 so I just reverted the version back